File: JavaScript/7 kyu/7 kyu - The highest profit wins.js
// https://www.codewars.com/kata/the-highest-profit-wins/train/python
// My solution
function minMax(arr){
    return [Math.min.apply(null,arr), Math.max.apply(null,arr)];
}

// ...
const R = require("ramda");

const minMax = (args) => R.juxt([Math.min, Math.max])(args[0], ...args);

// ...
const R = require("ramda");

const minMax = R.ifElse(
  R.pipe(R.length, R.equals(1)),
  R.ap(R.concat, R.identity),
  R.apply(R.juxt([Math.min, Math.max]))
);

// ...
function minMax(arr) {
    return [Math.min(...arr), Math.max(...arr)];
}

File: Python/7 - kyu/7 kyu - The highest profit wins.py
# https://www.codewars.com/kata/the-highest-profit-wins/train/python
# My solution
def min_max(lst):
    minimo = lst[0]
    maximo = lst[0]
    for i in lst:
        if i < minimo:
            minimo = i
        elif i > maximo:
            maximo = i
    return [minimo, maximo]

# ...
def min_max(lst):
  return [min(lst), max(lst)]

